src/Microsoft.Azure.EventHubs/EventHubClient.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.Azure.EventHubs
     using System.Threading.Tasks;
     using Microsoft.Azure.EventHubs.Amqp;
     using Microsoft.IdentityModel.Clients.ActiveDirectory;
+    using System.Linq;
+    using Microsoft.Azure.EventHubs.Core;
```

Comment:
Can you sort usings? Same for the other classes.

Reply:
Sure ! :)